Makefile

```
@@ -37,6 +32,19 @@ docker_build: ## Build Docker file
 		-t ${DOCKER_REPO}:${DOCKER_VERSION} \
 		.
 
+build_dev: ## Build development Docker file
+	docker build \
+		-f ./Dockerfile \
+		--target dev \
+		-t ${DOCKER_REPO}:dev \
+		.
+
+sh: build_dev ## Shell into development container
+	docker run -it ${DOCKER_REPO}:dev /bin/ash
+
+run: build ## Run in docker container
+	docker run -p 9540:9540 ${DOCKER_REPO}:${DOCKER_VERSION}
```

```suggestion
	docker run -p --rm 9540:9540 ${DOCKER_REPO}:${DOCKER_VERSION}
```

either `--rm` or `--name` to stop container bloat?